core/pom.xml

```diff
@@ -222,6 +222,7 @@
                         <java.net.preferIPv6Addresses>${test.ipv6}</java.net.preferIPv6Addresses>
                         <sun.net.useExclusiveBind>false</sun.net.useExclusiveBind>
                         <org.wildfly.openssl.path>${org.wildfly.openssl.path}</org.wildfly.openssl.path>
+                        <io.undertow.protocols.alpn.jdk8>true</io.undertow.protocols.alpn.jdk8>
```

Comment:
This doesn't look right to me. This will attempt to use the JDK's ALPN version which for Java 8 throws an {{UnsupportedOperationException}}.